docs/input-manifest/input-manifest.md

```diff
@@ -26,29 +26,30 @@ Manifest is a definition of the user's infrastructure. It contains cloud provide
 
 ## Providers 
 
-Contains configurations for different supported cloud providers. Atleast one provider needs to be defined out of the following supported providers 
+Contains configurations for supported cloud providers. At least one provider
+needs to be defined.
 
 - `gcp` [GCP](#gcp)
   
-  List of GCP configurations for [Google cloud](https://cloud.google.com/). This field is optional.
+  List of configurations for [Google Cloud ](https://cloud.google.com/). This field is optional.
```

Comment:
Might as well be `List of configuration options` like the rest.

Plus, you've added an extra whitespace after `Cloud`

Reply:
must have slipped through somehow, thanks.